### ISSUE TITLE
Flavors

### ISSUE THREAD

**Issue by DylanMoriarty** [kkowalsky/CAKEMAP#1]
I propose we do not one, but two versions of the Cake Map: Vanilla & Chocolate (ala' Mapbox Light & Dark).

With, of, course, other food maps to follow. Thoughts?

**Comment by vcj** [comment]
Yes, I am emphatically pro-that. Previous cake was just regular funfetti
from a box (frosting is also store bought funfetti) but I will make the
chocolate one using good stuff. For de dark map, though, the frosting might
not stand out. Perhaps a raspberry jam?

**Comment by vcj** [comment]
for _de_ dark map

learn how to type, johnson.

**Comment by DylanMoriarty** [comment]
I think the funfetti would work well for a vanilla version.

Also, Ooo. Rasberry Jam for the rivers or a dark red velvet would be niiiiiiiice.

![cococake](http://pizzazzerie.com/wp-content/uploads/2013/03/red-velvet-oreo-truffle-fudge-cake.jpg)

I'll work on making a pattern of chocolate chips as well. Might be able to use that for hillshade perhaps.

**Comment by vcj** [comment]
I scanned the frosting on a sheet of transparency plastic, but I tried to
get some of the texture with saran wrap (a couple pages in on the attached)
and it didn't really come out as expected but the reflection from the scan
light gives it a sort of hillshady feel. So it's an option if you want to
show roughness (accuracy notwithstanding).

**Comment by kkowalsky** [comment]
I agree with dylan! dark & light versions are silly

**Comment by kkowalsky** [comment]
(therefore great)
